apps/evm/src/pages/Account/NewPage/Transactions/TransactionRow/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { cn } from '@venusprotocol/ui';
+import { type TransactionWithAsset, TxType } from 'clients/api';
+import { Icon, type IconName, TokenIcon } from 'components';
+import { format } from 'date-fns';
+import { useTranslation } from 'libs/translations';
+import { useMemo } from 'react';
+import { generateExplorerUrl } from 'utilities';
+import { TransactionDetails } from '../TransactionDetails';
+
+export interface TransactionRowProps {
+  transactionData: TransactionWithAsset;
+  className?: string;
+}
+
+const getTransactionIcon = (txType: TxType): IconName => {
+  switch (txType) {
+    case TxType.Mint:
+    case TxType.Repay:
+      return 'transactionIn';
+    case TxType.Borrow:
+    case TxType.Redeem:
+      return 'transactionOut';
+    default:
+      return 'transactionCollateral';
+  }
+};
+
+export const TransactionRow: React.FC<TransactionRowProps> = ({ transactionData, className }) => {
+  const { transaction, asset } = transactionData;
+  const { underlyingToken } = asset.vToken;
+  const { t } = useTranslation();
+
+  const transactionTitle = useMemo(() => {
+    switch (transaction.txType) {
+      case TxType.Mint:
+        return t('account.transactions.txType.mint');
+      case TxType.Repay:
+        return t('account.transactions.txType.repay');
+      case TxType.Borrow:
+        return t('account.transactions.txType.borrow');
+      case TxType.Redeem:
+        return t('account.transactions.txType.redeem');
+      case TxType.Approve:
+        return t('account.transactions.txType.approve');
+      case TxType.ExitMarket:
+        return t('account.transactions.txType.exitMarket');
+      default:
+        return t('account.transactions.txType.enterMarket');
+    }
+  }, [t, transaction]);
+
+  return (
+    <li
+      className={cn(
+        'flex flex-col px-4 py-3 md:p-0 bg-cards rounded-xl md:rounded-none w-full md:flex-row items-center justify-evenly',
+        className,
+      )}
+    >
```

Comment:
On md and up, this component has fixed height of 64px and a hover effect assigned to it:
```suggestion
    <li
      className={cn(
        'flex flex-col px-4 py-3 bg-cards rounded-xl  w-full items-center justify-evenly md:flex-row md:p-0 md:h-16 md:hover:bg-lightGrey md:rounded-none',
        className,
      )}
    >
```